test/common/util.go

```diff
@@ -0,0 +1,8 @@
+package common
+
+// TestLogger defines operations common across different types of testing
+type TestLogger interface {
```

Comment:
Were you intending to move this from ``pkg/federatedtypes/crudtester`` or have it exist in both places?

Reply:
I was intending to move it but if I remember correctly, when I tried, there were other completely unrelated things that depended on it as well and I didn't want to break them. I will check this again.

Reply:
responded to this in latest commit